### PR TITLE
Fixed typo and bump version

### PR DESCRIPTION
**Pull request by po5i**
Hey @kaedroho, thanks for making this a pip package. Looking good but I found a small issue.

I've fixed this on this PR and bump the version to `1.0.1`.

### CODE REVIEW

File: wagtailapi_legacy/v1/apps.py [hunk 1]
```diff
@@ -6,15 +6,15 @@
 
 
 class WagtailAPIAppConfig(AppConfig):
-    name = 'wagtailapi_legacy_v1'
-    label = 'wagtailapi_legacy_v1'
+    name = 'wagtailapi_legacy.v1'
+    label = 'wagtailapi_legacy.v1'
```
Comment (kaedroho) [inline wagtailapi_legacy/v1/apps.py]: I think the ``label`` should still be an underscore, see https://github.com/wagtail/wagtail/blob/master/wagtail/api/v2/apps.py#L9

Comment (po5i) [reply wagtailapi_legacy/v1/apps.py]: Makes sense, updated!

**Comment by po5i** [comment]
Fixed, wdyt?

**Comment by kaedroho** [comment]
Apologies for the late response, I've been on holiday for the last two weeks.

This looks good to me, thanks!

**Comment by kaedroho** [comment]
Pushed to pypi

**Pull request merged**